monero-wallet-generator.html

```diff
@@ -11085,6 +11085,9 @@ <h5 id="gpg-instructions">How to verify GPG signatures</h5>
 generating = false;
 function genwallet_prefix()
 {
+  if(keys && !confirm('Are you sure? This wallet cannot be recovered once a new wallet is generated.')) {
+    return;
```

Comment:
I'm not sure I really understand this patch yet, but is this line missing "oldSelectionIndex = true;" ?

Reply:
No, "oldSelectionIndex = true;" is not missing here.
in fact these two lines are not tightly linked with the description (Check which coin is selected ...).
when brendanairvuz pushed https://github.com/moneromooo-monero/monero-wallet-generator/commit/f44d42cb088511ec02ec681a11882052ee1b0fcf he only checked the genwallet function and not the genwallet_prefix function. these two lines rectify this oversight.
To be more clear, these lines should have been on a separate commit.
I can do it if needed.